en/04_Changelogs/5.0.0.md

```diff
@@ -36,6 +36,7 @@ This changelog provides a list of changes between Silverstripe CMS 4 and 5.
   - [Email](#api-email)
   - [ORM](#api-orm)
   - [Dynamic properties](#dynamic-properties)
+  - [Templates](#templates)
```

Comment:
Someone said somewhere we don't need the specific anchor when it's the same as the one produced from the title. Though I actually prefer having it even when it's the same, for consistency, and for the cases when the title extends, we don't need to change it in several places.

Reply:
Is this meant as a comment on the _heading_ (where it's not needed)? It's always needed for the link, because you have to tell it where to link to.

Reply:
Ah, right, makes sense, I mixed those two together. All good.